ci/jenkins/test.sh

```diff
@@ -999,14 +992,16 @@ fi
 
 trap clean_antrea EXIT
 if [[ ${TESTCASE} =~ "windows" ]]; then
-    if [[ ${TESTCASE} =~ "containerd" ]]; then
+    if [[ ${TESTCASE} == *"userspace"* ]]; then
+        WINDOWS_YAML_SUFFIX="windows-containerd"
```

Comment:
I can only see yaml names like `antrea-windows-containerd*`, I suppose we should change the default value of WINDOWS_YAML_SUFFIX to "windows-containerd"? cc @XinShuYang

Reply:
Yes, the default value was shared by docker testbed, which has been deprecated.